docs/source/antennas_iq.rst

```diff
@@ -96,13 +96,17 @@ The file fields in the antennas_iq array files are:
 +-----------------------------------+---------------------------------------------+
 | | **cfs_freq**                    | | Frequencies sampled during the clear      |
 | | *float32*                       | | frequency search centered around          |
-| | [] or [num_cfs_samps]           | | baseband. The number of cfs samples is    |
-| |                                 | | the rx rate / the decimation stages.      |
+| | [] or [num_freqs]               | | baseband.                                 |
++-----------------------------------+---------------------------------------------+
+| | **cfs_masks**                   | | Masks for each cfs slice indicating which |
+| | *bool*                          | | frequencies cannot be selected for the    |
+| | [] or [num_cfs_slices,          | | slice tx freq.                            |
```

Comment:
The mask for each slice will be contained with only the data file for that slice, so we don't need a `num_cfs_slices` dimension

Reply:
I do think the data file formatting for the CFS parameters needs a review later. When we touch on the data structure later we can ensure it is set up correctly, but I'll make these changes now.